test/CollapsibleNavSpec.js

```diff
@@ -111,4 +112,30 @@ describe('CollapsibleNav', function () {
         , idx = classArray.indexOf('navbar-collapse');
     assert.equal(classArray.indexOf('navbar-collapse', idx+1), -1);
   });
+
+  it('Should not warn about deprecation when collaps_i_ble property is used', function () {
+    let Component = React.createClass({
+      render: function() {
+        return (
+          <CollapsibleNav collapsible />
+        );
+      }
+    });
+    ReactTestUtils.renderIntoDocument(<Component />);
+
+    console.warn.called.should.be.false;
+  });
+
+  it('Should warn about deprecation when collaps_a_ble property is used', function () {
+    let Component = React.createClass({
+      render: function() {
+        return (
+          <CollapsibleNav collapsable />
+        );
+      }
+    });
+    ReactTestUtils.renderIntoDocument(<Component />);
+
+    shouldWarn('deprecated');
+  });
```

Comment:
What are your thoughts on having both these tests in the CollapsableNavSpec.js file, so it's as simple as deleting the file when we remove the old functionality?